src/components/shop-tabs-overlay.js

```diff
@@ -43,15 +43,14 @@ class ShopTabsOverlay extends LitElement {
     this._transitionsInFlight = [];
   }
 
-  _firstRendered() {
+  finishFirstUpdate() {
     this.addEventListener('transitionend', (e)=>this._onTransitionend(e));
   }
 
-  _propertiesChanged(props, changed, oldProps) {
-    if (changed && 'target' in changed) {
-      this._targetChanged(props.target, oldProps.target);
+  finishUpdate(changedProps) {
```

Comment:
@sorvell In IE the given `changeProps` arg is not initialized correctly (never has `target`).